docs/email/postfix/configure-spf-and-dkim-in-postfix-on-debian-8.md

```diff
@@ -413,3 +434,5 @@ The reason the YYYYMM format is used for the selector is that best practice call
     Make sure they both start without any errors.
 
 7.  After a couple of weeks, all email in transit should either have been delivered or bounced and the old DKIM key information in DNS won't be needed anymore. Delete the old `YYYYMM._domainkey` TXT records in each of your domains, leaving just the newest ones (most recent year and month). Don't worry if you forget and leave the old keys around longer than planned. There's no security issue. Removing the obsolete records is more a matter of keeping things neat and tidy than anything else.
+
+##
```

Comment:
It mentions the software, but not installing it. The instructions here are for the DNS records needed for outbound mail (allowing other mail servers to check your mail for conformance to your DMARC policies). I haven't written up the section on hooking the OpenDMARC daemon into Postfix for inbound mail (your mailserver checking incoming messages for conformance to their domain's DMARC policies) and generating aggregate reports. When I add that section I'll update the first section on installing required software to include OpenDMARC.